src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/GetAzureRmAnalysisServicesServerGateway.cs

```diff
@@ -0,0 +1,70 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.AnalysisServices.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Analysis.Models;
+
+namespace Microsoft.Azure.Commands.AnalysisServices
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmAnalysisServicesGateway"),
+        OutputType(typeof(GatewayInfo))]
+    [Alias("Get-AzureAsGateway")]
+    public class GetAzureAnalysisServicesServerGateway: AnalysisServicesCmdletBase
+    {
+        [Parameter(Position = 0,
```

Comment:
This needs to follow the cmdlet review suggestions: https://github.com/Azure/azure-powershell/wiki/Azure-Analysis-Services-Gateway-Operations#review-feedback

Reply:
Already send email to ask for details. No idea what you mean, please be more specific.

Reply:
Please read the information from the link above.